config/routes.rb

```diff
@@ -0,0 +1,14 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  root 'works#main'
+
+  resources :users, only: [:index, :show]
+  resources :sessions
```

Comment:
I'm not sure if you need the full `resources` for `:sessions` here, especially since you've got login/logout listed explicitly below.